src/init.cpp

```diff
@@ -157,7 +157,6 @@ NODISCARD static bool CreatePidFile()
 static std::unique_ptr<ECCVerifyHandle> globalVerifyHandle;
 
 static boost::thread_group threadGroup;
-static CScheduler scheduler;
 
 void Interrupt(NodeContext& node)
 {
```

Comment:
style-nit in commit 047c1fd:


It should be safe and a bit cleaner to reset the scheduler unique_ptr at the end of `void Shutdown(NodeContext& node)`.